### PR TITLE
Add a block attributes filter

### PR DESCRIPTION
**Pull request by lukecarbis**
cc. @phpbits

### CODE REVIEW

**Comment by lukecarbis** [comment]
@kienstra This is a real quick win, if you could please review?

File: php/blocks/class-loader.php [hunk 1]
```diff
@@ -187,7 +187,7 @@ public function get_block_attributes( $block ) {
 			}
 		}
 
-		return $attributes;
+		return apply_filters( 'block_lab_get_block_attributes', $attributes );
```
Comment (kienstra) [inline php/blocks/class-loader.php]: @lukecarbis, would you mind if I committed a DocBlock for `apply_filters` here, including a `@param` tag for `$attributes`?

Comment (lukecarbis) [reply php/blocks/class-loader.php]: Of course! I should have thought of that. Go for it.

Comment (kienstra) [reply php/blocks/class-loader.php]: Thanks!

**Comment by kienstra** [comment]
**Passing `$block` to filter?**

Hi @lukecarbis, 
what do you think about https://github.com/getblocklab/block-lab/pull/363/commits/3c335e49d477a9bd5e55f163a40a09c8cd3b5217, which also passes `$block` to the filter?

Otherwise, it might be hard to know which block it's for:

attributes-value

**Comment by kienstra** [comment]
Also, I'm working on the failed Travis build in #364.

**Comment by lukecarbis** [comment]
@kienstra Thanks for those two commits. They both look great! Especially 3c335e49d477a9bd5e55f163a40a09c8cd3b5217 – good thinking. Ready for you to re-review.

**Review by kienstra — APPROVED** [review]
**Approved**

Thanks, @lukecarbis! This should be ready to merge.

**Pull request merged**